samples/src/main/java/cli/PaymentInput.java

```diff
@@ -0,0 +1,14 @@
+package cli;
+
+public class PaymentInput {
```

Comment:
If you are using JDK 21 this could be a Record I think

Reply:
It could, but I'm not convinced it should...